js/adapt-contrib-resourcesHelpers.js

```diff
@@ -1,53 +1,36 @@
-define(function(require) {
-
-	var Handlebars = require('handlebars');
-
-	Handlebars.registerHelper('if_collection_contains', function(collection, attribute, value, block) {
-		var makeBlockVisible = false;
-
-		_.each(collection, function(resource) {
-			if (resource[attribute] === value) {
-				makeBlockVisible = true;
-			}
-		});
-		if(makeBlockVisible) {
-            return block.fn(this);
-        } else {
-            return block.inverse();
-        }
-    });
-
-    Handlebars.registerHelper('if_collection_contains_only_one_item', function(collection, attribute, block) {
-		var attributeCount = [];
-
-		_.each(collection, function(resource) {
-			var resourceAttribute = resource[attribute];
-			if (_.indexOf(attributeCount, resourceAttribute) === -1) {
-				attributeCount.push(resourceAttribute);
-			}
-		});
-
-		if (attributeCount.length <= 1) {
-			return block.fn(this);
-		} else {
+define([
+	'handlebars',
+	'core/js/adapt'
+], function(Handlebars, Adapt) {
+
+	Handlebars.registerHelper('if_has_type', function(resources, type, block) {
```

Comment:
Could you change these to reflect the method used in [src/core/js/helper.js](https://github.com/adaptlearning/adapt_framework/blob/master/src/core/js/helpers.js#L210-L214)?

It's much easier to read having an object:
```js
var helpers = {
  if_has_type: function(resources, type, block) { },
  if_has_multiple_types: function(resources, type, block) { }
};
for (var name in helpers) {
  if (!helpers.hasOwnProperty(name)) continue;
  Handlebars.registerHelper(name, helpers[name]);
}
```

And also prefix the helpers with `"resources_"` so that it's clear that these helpers are not global?